pkg/controller/operators/olm/operator.go

```diff
@@ -124,6 +127,7 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 		client:                config.externalClient,
 		ogQueueSet:            queueinformer.NewEmptyResourceQueueSet(),
 		csvQueueSet:           queueinformer.NewEmptyResourceQueueSet(),
+		olmConfigQueue:        workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "olmConfig"),
```

Comment:
What was the rationale again for this queue implementation vs. using the queueinformer?

Reply:
I believe that you're asking why this field is not set the same way as the others using the `queueinformer.NewEmptyResourceQueueSet()` function.

The QueueSets eventually use the same `workqueue.NewNamedRateLimitingQueue` func to add the queue to the set ([example](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/operators/olm/operator.go#L158-L159). OLM only ever cares about the clusterScoped OLMConfig queue so I didn't see the need to include a set.

Reply:
Following up here, I attempted to use a set because I am not a fan of getting the olmConfig resource to requeue it within the [SyncCopyCSV](https://github.com/operator-framework/operator-lifecycle-manager/pull/2466/files#diff-1c45679686ec7da9a4e3c02d009d84f465130fec2278b9512d27d53e23224f3aR1366) function, but the set didn't seem to actually requeue the olmConfig resource. We should probably look into this in the future.